### PR TITLE
[11.0][MIG] l10n_nl_bank

### PR DESCRIPTION
**Pull request by astirpe**

### CODE REVIEW

**Review by CasVissers-360ERP — APPROVED** [review]

File: l10n_nl_bank/readme/CREDITS.rst [hunk 1]
```diff
@@ -0,0 +1,4 @@
+Images
+------
+
+* Bank image: <http://www.iconninja.com/banking-icon-466530>
```
Comment (hbrunn) [inline l10n_nl_bank/readme/CREDITS.rst]: I can't find license information on this website. Maybe better use the opportunity to get a creative commons one from [the noun project](https://thenounproject.com/search/?q=bank)?

**Review by hbrunn — APPROVED** [review]
rest looks good

File: l10n_nl_bank/readme/CREDITS.rst (hunk 1 above)
Comment (astirpe) [reply l10n_nl_bank/readme/CREDITS.rst]: Agree

**Review by CasVissers-360ERP — APPROVED** [review]

File: l10n_nl_bank/readme/CREDITS.rst (hunk 1 above)
Comment (hbrunn) [reply l10n_nl_bank/readme/CREDITS.rst]: thanks!

**Comment by astirpe** [comment]
Commits squashed

**Review by CasVissers-360ERP — APPROVED** [review]
LGTM

**Review by yung-wang — APPROVED** [review]
Works fine here.

**Pull request merged**